<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortWidget</class>
 <widget class="QWidget" name="PortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,1">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="interfaceSelector"/>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="text">
      <string>Open port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Close port</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
